packages/locales/lib/human/de.json

```diff
@@ -704,5 +704,35 @@
   "gender_filters_all": "Geschlecht gilt für alle",
   "and_caption": "miteinander verbunden",
   "or_caption": "filtert unabhängig",
-  "filter_help": "Filter Hilfe"
+  "filter_help": "Filter Hilfe",
+  "kilometers": "Kilometer",
+  "miles": "Miles",
+  "distance_unit": "Entfernungseinheit",
+  "data_management": "Einstellungsverwaltung",
```

Comment:
Datenverwaltung

Or do you think it's more intuitive to name it like you said?